app/config.py

```diff
@@ -539,7 +541,7 @@ class Live(Config):
     CRONITOR_ENABLED = True
 
     ENABLED_CBCS = {BroadcastProvider.THREE, BroadcastProvider.O2, BroadcastProvider.VODAFONE}
-    DVLA_EMAIL_ADDRESS = os.getenv('DVLA_EMAIL_ADDRESS')
+    DVLA_EMAIL_ADDRESSES = os.getenv('DVLA_EMAIL_ADDRESSES')
```

Comment:
you'll need to call `json.loads` here (as we do elsewhere). and set a default val of `[]` probably.

also, can this be deduplicated by moving it from `Live` to the default config? that'll make it consistent with how we load all the other environment variables

Reply:
good catch - done.